### PR TITLE
support hostport filed for kafka

### PR DESCRIPTION
**Pull request by mrtc0**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

> /kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area falco-chart

> /area falco-exporter-chart

/area falcosidekick-chart

**What this PR does / why we need it**:

Kafka output settings work with `Kafka.HostPort` instead of `Kafka.URL` .  
This PR works by merging https://github.com/falcosecurity/falcosidekick/pull/194 .

**Which issue(s) this PR fixes**:

https://github.com/falcosecurity/falcosidekick/pull/194

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [ ] Chart Version bumped
- [x] Variables are documented in the README.md
- [ ] CHANGELOG.md updated

### CODE REVIEW

**Review by Issif — COMMENTED** [review]
You need to bump up the chart version and update the changelog please

File: falcosidekick/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.2.8
+
+### Fixes
+
+* FIxed to specify `kafka.hostPort` insted of `kafka.url`
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:12]:
```suggestion
* FIxed to specify `kafka.hostPort` instead of `kafka.url`
```

**Review by Issif — CHANGES_REQUESTED** [review]

**Comment by Issif** [comment]
You need to signoff you commit please 🙏

File: falcosidekick/CHANGELOG.md (hunk 1 above)
Comment (mrtc0) [reply falcosidekick/CHANGELOG.md]: Sorry, I fixed it.

**Review by Issif — APPROVED** [review]
it's ok now, thanks.

can you create a PR for bumping up the dependency in falco chart, please https://github.com/falcosecurity/charts/blob/e75e690537b665f2da42da0cbb88ab8329f1821e/falco/Chart.yaml#L22

**Pull request merged**